src/main/java/Duke.java

```diff
@@ -1,10 +1,178 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws DukeException, IOException {
+        System.out.println("Hello! I'm Duke\n" +
+                " What can I do for you?");
+        Scanner sc = new Scanner(System.in);
+        int exit = 0;
+        ArrayList<Activities> toDoList = new ArrayList<>();
+        int totalActivities = 0;
+        File f = new File("List.txt");
+        try {
+            if (!f.exists()) {
+                f.createNewFile();
+            }
+        }catch(IOException e){
+            System.out.println("sorry this file already exists");
+        }
+        Scanner fileScanner = new Scanner(f);
+        while(fileScanner.hasNext()) {
+            String previousTasks = fileScanner.nextLine();
+            int temp = Integer.parseInt(previousTasks.split(" ", 2)[0]);
+            previousTasks = previousTasks.split(" ", 2)[1];
+            switch (previousTasks.split(" ")[0]) {
+            case "todo":
+                Activities temp0 = new Todo(previousTasks.split(" ", 2)[1]); // temp is an instance of todos
```

Comment:
Please consider changing the name of temp0 to something like "todoItem" for readability.